pkg/profiler/cpu/cpu.go

```diff
@@ -301,7 +365,21 @@ func (p *CPU) addUnwindTableForProcess(pid int) {
 
 	level.Debug(p.logger).Log("msg", "adding unwind tables", "pid", pid)
 
-	err = p.bpfMaps.addUnwindTableForProcess(pid, nil, true)
+	// @nocommit: Improve.
+	procInfo, err := p.processInfoManager.Fetch(context.TODO(), pid)
```

Comment:
Probably it should get the context from the caller. Or consider passing down the process info from the profiler.